mobile/profile.go

```diff
@@ -45,12 +45,3 @@ func (m *Mobile) Avatar() (string, error) {
 
 	return m.node.Avatar(), nil
 }
-
-// SetAvatar calls core SetAvatar
-func (m *Mobile) SetAvatar(hash string) error {
```

Comment:
This is removed because to set the avatar on mobile, you just have to add a file to the account thread (the mobile API handles the announcement by detecting the thread is the account thread). In the future, a `setAvatar` method could be added to the iOS/Android/RN SDKs which would just do that behind the scenes.